contrib/hack/test-apiserver.sh

```diff
@@ -37,5 +50,4 @@ kubectl delete -f contrib/examples/apiserver/broker.yaml
 kubectl delete -f contrib/examples/apiserver/serviceclass.yaml
 kubectl delete -f contrib/examples/apiserver/instance.yaml
 kubectl delete -f contrib/examples/apiserver/binding.yaml
-
-stop-server.sh
+set +x
```

Comment:
I like bookends :-)

This way if someone adds more logic later I've already "cleaned up" for them.